users.js

```diff
@@ -1355,6 +1355,10 @@ class User {
 			// 10 seconds ago
 			return false;
 		}
+		if (this === user) {
+			this.popup("You can't battle your own account. Please use something like Private Browsing to battle yourself.");
```

Comment:
Should probably only be implemented in one place, not both. I think in `/challenge` is enough.